packages/terra-table/src/subcomponents/Section.jsx

```diff
@@ -162,37 +168,118 @@ function Section(props) {
     onSectionSelect(id);
   }, [id, onSectionSelect]);
 
+  const sectionHeader = (
+    <tbody>
+      {/* Render section rows */}
+      {!isHidden && (
+      <tr
+        aria-rowindex={sectionRowIndex}
+        className={cx('header-row', theme.className)}
+        data-section-id={id}
+      >
+        <th
+          id={`${tableId}-${id}`}
+          className={cx('header-cell')}
+          align="left"
+          colSpan={displayedColumns.length}
+          role="columnheader"
+          scope="col"
+          tabIndex={isGridContext && !hasSectionButton ? -1 : undefined}
+        >
+          <SectionHeader
+            className={cx('section-header')}
+            text={text}
+            isOpen={hasSectionButton ? !isCollapsed : undefined}
+            onClick={hasSectionButton ? handleClick : undefined}
+            isTitleSticky
+            boundedWidth={boundingWidth}
+          />
+        </th>
+      </tr>
+      )}
+    </tbody>
+  );
+
+  const rowProps = {
+    displayedColumns,
+    firstRowId,
+    height: rowHeight,
+    isTableStriped,
+    lastRowId,
+    onCellSelect,
+    rowHeaderIndex,
+    rowMinimumHeight,
+    rowSelectionMode,
+    sectionId: !isHidden ? id : undefined,
+    tableId,
+  };
+
+  if (subsections) {
```

Comment:
What happens if the section has its own rows as well as subsections?  I don't believe that use case is covered in this design.

Reply:
Subsections definitely can have rows. See screen shot.
![image](https://github.com/cerner/terra-framework/assets/84464759/6de4c735-e11e-45f4-831f-4c2de4492c99)

Reply:
Here is a link to a video that shows the full workflow: 
https://oracle-my.sharepoint.com/:v:/r/personal/sriraksha_k_oracle_com/Documents/Orders%20for%20Signature%20Accessibility/Routing%20Section%20issue%20in%20OFS.mov?csf=1&web=1&e=pJAyt6&nav=eyJwbGF5YmFja09wdGlvbnMiOnt9LCJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbE1vZGUiOiJtaXMiLCJyZWZlcnJhbFZpZXciOiJwb3N0cm9sbC1jb3B5bGluayIsInJlZmVycmFsUGxheWJhY2tTZXNzaW9uSWQiOiI4MmMxMWE2ZC02YjQ3LTQ5MTgtOTRhZC1lNzViYWU5YzUxYTUifX0%3D

Reply:
The subsections are used to show the routing of medication orders. There is typically a row under the subsection for each medication that qualifies for that route; e.g., electronic prescription, pharmacy call, etc.

Reply:
Oh I meant if section level row and subsections are both simultaneously possible?

e.g.
- section header
  - section row
  -  section row
  - subsection
    - subsection row
    - subsection row 2

Reply:
I will have to reach out the Clinical product manager for that question.